asserts/snapasserts/validation_sets.go

```diff
@@ -29,6 +29,13 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+// InstalledSnap holds the minimal details about an installed snap required to
+// check the validation set.
+type InstalledSnap struct {
+	SnapID string
```

Comment:
what should we do about locally installed snaps, they should still fail invalid or revision constraints, but succeed required only without revision?